### PR TITLE
added time param in update_reference_from_subscribers()

### PR DESCRIPTION
**Pull request by Robotgir**
to calculate ageOfLastCommand and use reference_timeout variable in chainable and standard controller setup a time param is required inupdate_reference_from_subscribers(
  const rclcpp::Time & time, const rclcpp::Duration & /*period*/) as in update_and_write_commands(
  const rclcpp::Time & time, const rclcpp::Duration & /*period*/)

### CODE REVIEW

**Comment by anfemosa** [comment]
In my opinion, the .vscode configuration files should not be included in the merge request

**Comment by Robotgir** [comment]
@anfemosa Thankyou for the feedback. It makes sense. I will remove them.

**Review by bmagyar — APPROVED** [review]
Looks good, thanks!

**Review by destogl — APPROVED** [review]

**Comment by saikishor** [comment]
Looks good 👏🏽

**Review by bmagyar — APPROVED** [review]

File: controller_manager/test/test_chainable_controller/test_chainable_controller.hpp [hunk 1]
```diff
@@ -63,7 +63,8 @@ class TestChainableController : public controller_interface::ChainableController
   CONTROLLER_MANAGER_PUBLIC
   std::vector<hardware_interface::CommandInterface> on_export_reference_interfaces() override;
 
-  controller_interface::return_type update_reference_from_subscribers() override;
+  controller_interface::return_type update_reference_from_subscribers(
+    const rclcpp::Time & /*time*/, const rclcpp::Duration & /*period*/) override;
```
Comment (destogl) [inline controller_manager/test/test_chainable_controller/test_chainable_controller.hpp:67]:
```suggestion
    const rclcpp::Time & time, const rclcpp::Duration & period) override;
```

**Review by destogl — APPROVED** [review]
Thanks!

**Review by bmagyar — APPROVED** [review]

**Pull request merged**